src/components/DogList.jsx

```diff
@@ -1,15 +1,57 @@
+import PropTypes from 'prop-types';
 import Dog from './Dog';
 import './DogList.css';
 
-const DogList = () => {
-  return  (
-    <div>
-      <h2 className='dogListHeader'> Dog List </h2>
-      <Dog />
-      <Dog />
-      <Dog />
-    </div>
-)
-};
+const DogList = (props) => {
+    /* 
 
+    We should be sure ot focus on the when the code here runs in the lifecycle of the component.
+    Also, we should be sure to discuss that there are alternatives approaches to generating 
+    a list of components.
+    
+    */
+    
+    const getDogListJSX = (dogs) => {
+      return dogs.map((dog) => {
+        return (
+          <Dog
+            key={dog.id}
+            id={dog.id}
+            name={dog.name}
+            breed={dog.breed}
+            age={dog.age}
+            chip={dog.chip}
+          />
+        );
+      });
+    };
+
+    return <ul className="dogs__list">{getDogListJSX(props.dogs)}</ul>;
```

Comment:
Let's settle on one style of class name. In the previous branch it was `dogList`, but here it's `dogs__list`. Personally, I'm not a big fan of BEM-style classes, so would prefer `dogList`, but as long as we're consistent I'd be happy.

Reply:
Was the removal of `<h2> Dog List </h2>` intentional? We still have the header styling. If the heading was intended to be removed, we should also remove the style rule. If we keep the heading, then my feedback on where to stick the class name applies here as well. It should be on the outer element (we'd need to bring back the div to group the heading and list).